3rdParty/iresearch/core/analysis/analyzers.hpp

```diff
@@ -118,11 +118,16 @@ class IRESEARCH_API analyzers {
 
   ////////////////////////////////////////////////////////////////////////////////
   /// @brief for static lib reference all known scorers in lib
-  ///        for shared lib NOOP
   ///        no explicit call of fn is required, existence of fn is sufficient
   ////////////////////////////////////////////////////////////////////////////////
   static void init();
 
+  ////////////////////////////////////////////////////////////////////////////////
+  /// @brief for static lib reference all known scorers in lib
+  ///        no explicit call of fn is required, existence of fn is sufficient
+  ////////////////////////////////////////////////////////////////////////////////
+  static void init_with_model();
```

Comment:
that'd break iresearch tests, instead I suggest we pass a visitor to `init()` to approve analyzers we'd like to use